Parsers/ASimRegistryEvent/test/TrendMicro_VisionOne_vimRegistryEvent_DataTest.csv

```diff
@@ -0,0 +1,9 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 1 records (0.44%) for field [DvcHostname] of type [Hostname]: [""QA_test""] (Schema:RegistryEvent)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 1 records (0.44%) for field [Dvc] of type [Hostname]: [""QA_test""] (Schema:RegistryEvent)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 229 records (100.0%) for field [EventProduct] of type [Enumerated]: [""Vision One""] (Schema:RegistryEvent)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 229 records (100.0%) for field [EventType] of type [Enumerated]: [""RegistryValueSet""] (Schema:RegistryEvent)"
+"(1) Warning: Empty value in 1 records (0.44%)  in mandatory field [EventEndTime] (Schema:RegistryEvent)"
+"(1) Warning: Empty value in 1 records (0.44%)  in mandatory field [EventStartTime] (Schema:RegistryEvent)"
+"(2) Info: Empty value in 94 records (41.05%)  in recommended field [RegistryPreviousValue] (Schema:RegistryEvent)"
+"(2) Info: Empty value in 94 records (41.05%)  in recommended field [RegistryValue] (Schema:RegistryEvent)"
```

Comment:
After ingesting sample data. I was getting high % of mandatory and recommended fields. Please validate that in the logs your using, % of missing mandatory and recommeded are low

Reply:
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/c12531b0-d898-485f-b166-95c69fcd2a0f)

Reply:
Can you please ingest the logs again and confirm the above behavior stays? As we haven't provided data with 880 records.

Reply:
that's because I ingested same data multiple times.